src/dbt_client/index.ts

```diff
@@ -112,12 +113,18 @@ export class DBTClient implements Disposable {
       if (!this.shownError) {
         // We don't want to flood the user with errors
         this.shownError = true;
-        const answer = await window.showErrorMessage(
-          "Please ensure dbt is installed.",
-          DbtInstallationPromptAnswer.INSTALL,
-        );
-        if (answer === DbtInstallationPromptAnswer.INSTALL) {
-          commands.executeCommand("dbtPowerUser.installDbt");
+        let message = "Please ensure dbt is installed.";
+        if (this.dbtIntegrationMode === "cloud") {
+          message = "Please ensure dbt cloud cli is installed.";
+          window.showErrorMessage("Please ensure dbt cloud cli is installed.");
+        } else {
+          const answer = await window.showErrorMessage(
+            "Please ensure dbt is installed.",
+            DbtInstallationPromptAnswer.INSTALL,
+          );
+          if (answer === DbtInstallationPromptAnswer.INSTALL) {
+            commands.executeCommand("dbtPowerUser.installDbt");
+          }
```

Comment:
I think we can comment this for now as dbt cannot yet be installed. Add a TODO

Reply:
currently in `master`, it shows `Install dbt` button in both cases. This shows the button only for core